contentcuration/contentcuration/frontend/channelEdit/vuex/contentNode/actions.js

```diff
@@ -35,6 +35,13 @@ export function loadChildren(context, { parent, channel_id }) {
   });
 }
 
+export function loadAncestors(context, { id, channel_id }) {
+  let node = context.state.treeNodesMap[id];
+  return Tree.where({ max_lft: node.sort_order, min_rght: node.rght, channel_id }).then(nodes => {
+    return loadContentNodes(context, { ids: nodes.map(node => node.id) });
+  });
```

Comment:
just curious - is this a common pattern on the client-side? I would have expected the fact that MPTT is used to be an implementation detail of the server, hidden from the client.

I can understand how it might be useful to do this, but it also introduces coupling that can be difficult to recover from later.

Reply:
We'll add some ancestor information in a separate PR, but will require a bit more thought on how to handle best